README.md

```diff
@@ -72,3 +84,23 @@ npm run dev # at root
 
 It will open up a playground react app at `http://localhost:4000`.
 Make a `.env` file in the playground directory following the `.env[example]` file.
+
+## 🤔FAQ🤔
+
+#### How to set up?
+
+Part 1 - https://www.youtube.com/watch?v=l-6OIvolqyE
+
+Part 2 - https://www.youtube.com/watch?v=AspuaA36VsA
+
+Part 3 - https://www.youtube.com/watch?v=3PfEJypXXTQ
+
+#### Features
```

Comment:
Add something like, learn more about the roots of EmbeddedChat here and give the link

Reply:
Link of what?